### ISSUE TITLE
Multiple calls to SimpleServerFactory.configure()

### ISSUE THREAD

**Issue by dsKarthick** [dropwizard/dropwizard#2104]
When I run a dropwizard application, I see aforementioned [SimpleServerFactory.configure() ](http://bit.ly/2uaLo88) invoked a couple of times via
    
1. [EnvironmentCommand.java#L40](http://bit.ly/2uaGCY8)
2. [EnvironmentCommand.java#L44](http://bit.ly/2uaLZXq)

Following are the stack full stack traces for the paths leading up to [SimpleServerFactory.java#L141](http://bit.ly/2ub1FKm)

# Via  [EnvironmentCommand.java#L40](http://bit.ly/2uaGCY8)
```
"main"@1 in group "main": RUNNING
configure:141, SimpleServerFactory {io.dropwizard.server}
run:40, EnvironmentCommand {io.dropwizard.cli}
run:85, ConfiguredCommand {io.dropwizard.cli}
run:74, Cli {io.dropwizard.cli}
run:89, Application {io.dropwizard}
main:12, Main {io.myapp}
```

# Via [EnvironmentCommand.java#L44](http://bit.ly/2uaLZXq)

```
"main"@1 in group "main": RUNNING
configure:141, SimpleServerFactory {io.dropwizard.server}
build:103, SimpleServerFactory {io.dropwizard.server}
run:49, ServerCommand {io.dropwizard.cli}
run:44, EnvironmentCommand {io.dropwizard.cli}
run:85, ConfiguredCommand {io.dropwizard.cli}
run:74, Cli {io.dropwizard.cli}
run:89, Application {io.dropwizard}
main:12, Main {io.myapp}
```

Perhaps, remove [this](http://bit.ly/2uaGLec) line or include it only if the configure has not been run already?

**Comment by mattnelson** [comment]
Why are all the link through bitly? I didn't click on any of them, because it seems kinda phishy.

**Comment by mattnelson** [comment]
This looks like the issue that was resolved with #1965

**Comment by dsKarthick** [comment]
@mattnelson Thanks for pointing me to https://github.com/dropwizard/dropwizard/pull/1965. Maybe you are already aware but please note that https://github.com/dropwizard/dropwizard/pull/1965 fixes [DefaultServerFactory.java](https://github.com/dropwizard/dropwizard/blob/master/dropwizard-core/src/main/java/io/dropwizard/server/DefaultServerFactory.java), not [SimpleServerFactory.java](https://github.com/dropwizard/dropwizard/blob/master/dropwizard-core/src/main/java/io/dropwizard/server/SimpleServerFactory.java)
